docs/CONTRIBUTING.md

```diff
@@ -24,9 +24,9 @@ code running
 ### Chapter 1: Contributing to the DevC Pune repository
 <ins>Step 1: Fork **[devcpune/kaleidoscope](https://github.com/devcpune/kaleidoscope/tree/master)** repository</ins>
 
-<ins>Step 2: Copy project into the forked repo</ins>
+<ins>Step 2: Copy project into the forked repo under 'projects' folder</ins>
```

Comment:
adding a link to the projects folder would be useful.

Reply:
Will make the change.

Reply:
@AJV009  Do i need to make a new pull request for the changes requested? Since new to github, so not sure about the process.